dataeng/resources/tableau-backup.sh

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
+set -ex
+
+
+ssh -tt centos@$TABLEAU_SERVER_IP -o StrictHostKeyChecking=no "
+
+sudo -H -u root bash -c 'aws s3 cp /var/opt/tableau/tableau_server/data/tabsvc/files/backups/$(date '+%Y.%m.%d').11.00-tableau-data-backup.tsbak s3://edx-analytics-tableau/backups/$(date '+%Y.%m.%d').11.00-tableau-data-backup.tsbak'
+sudo -H -u root bash -c 'rm /var/opt/tableau/tableau_server/data/tabsvc/files/backups/$(date '+%Y.%m.%d').11.00-tableau-data-backup.tsbak'
```

Comment:
How much control over the names of backfup files do we have? I imagine we will be upgrading the version of tableau we use. One thing I find useful is to create a string for the name of the file you will be manipulating and use it as a variable. It makes things a bit more reabable.

Reply:
Updated the code. I am now creating an environment variable so we will have to change the name only place.